configserv/image.yaml

```diff
@@ -1,4 +1,4 @@
-version: 1.0
+version: latest
```

Comment:
Depends where do you build your images. If you will be using our CI (which is something I'm currently working on) - you get it for free :) But generally speaking - yes, you can add any tag after you build the image.